### PR TITLE
fix/#62/fallback empty values to show on console

### PR DESCRIPTION
**Pull request by ologbonowiwi**
- test: add test with None values on invitation
- feat: print empty line when the value is None

This PR handles issue #62, and add tests to confirm that it's really covered

### CODE REVIEW

File: inb/api/invitation/status.py [hunk 1]
```diff
@@ -117,8 +117,15 @@ def _replace_template_var_with_template_value(
...
     for replace_template_var_with_value_pair in replace_template_var_with:
-      message_template = message_template.replace(
-          *replace_template_var_with_value_pair)
+      # if the value to be replaced is not None
+      if replace_template_var_with_value_pair[1] is not None:
+        # then, replace the template variable with the value
+        message_template = message_template.replace(
+            *replace_template_var_with_value_pair)
+      else:
+        # provide an empty string as the valid value
```
Comment (joshiayush) [inline inb/api/invitation/status.py:126]: If `None` value is creating a problem then in that case these inline comments are not needed. The code itself is readily understandable, could you please remove them before I merge?

File: inb/tests/test_invitation_status.py [hunk 2]
```diff
@@ -113,6 +113,33 @@ def test_invitation_send_status_to_console(invitation):
...
+      name='John Smith',
+      occupation=None,
+      location=None,
+      profileid='john-smith',
+      profileurl='https://www.linkedin.com/in/john-smith',
+      status='sent',
+      elapsed_time=10.0)
+
+  expected_output = ('  ✔  John Smith\n'
+                      '  \n'
```
Comment (joshiayush) [inline inb/tests/test_invitation_status.py:130]: There's a problem in indentation here and in the following lines, could you please fix that?

File: inb/api/invitation/status.py [hunk 3]
```diff
@@ -117,8 +117,15 @@ def _replace_template_var_with_template_value(
...
+        message_template = message_template.replace(
+            replace_template_var_with_value_pair[0], '')
```
Comment (joshiayush) [inline inb/api/invitation/status.py:128]: Moreover, this entire `if` `else` ladder could be made more readable if you could just change the above code to:

```python
template_var = replace_template_var_with_value_pair[0]
template_val = replace_template_var_with_value_pair[1]

if template_val is None:
  ...
else
  ... 
```

Could you please make this change before I merge?

**Review by joshiayush — CHANGES_REQUESTED** [review]
I really appreciate the fact that you consider adding tests for your changes. Keep it up and thanks for helping in my projects.

File: inb/api/invitation/status.py (hunk 3 above)
Comment (joshiayush) [inline inb/api/invitation/status.py:128]: I would like to request one more change. Is there something else that we can display on console rather than empty values. Like some form of text or one word like `NaN` for better understanding? which will just tell the user that we couldn't find this one from the profile.

**Review by joshiayush — CHANGES_REQUESTED** [review]

File: inb/api/invitation/status.py (hunk 3 above)
Comment (ologbonowiwi) [reply inb/api/invitation/status.py:128]: If you have something in mind I'll be happy to implement it

Comment (joshiayush) [reply inb/api/invitation/status.py:128]: Use `NaN` for now
